lib/jekyll/plugin_manager.rb

```diff
@@ -27,7 +27,7 @@ def require_gems
       site.gems.each do |gem|
         if plugin_allowed?(gem)
           Jekyll.logger.debug("PluginManager:", "Requiring #{gem}")
-          require gem
+          Jekyll::External.require_with_graceful_fail([gem])
         end
```

Comment:
Nope, only a logger.error message about the Dependency Error in case of `LoadError` in `require_with_graceful_fail`. 

Think it would be valuable to include a Jekyll.logger.debug message about which gem is being required in `require_with_graceful_feedback`, provided that we remove any redundant logger.debug messages in places that use `require_with_graceful_fail`. And looking through the codebase, there don't seem to be any.